[2022-02-15 03:25:38,183] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.test.my_new_project.dim_customers manual__2022-02-15T03:24:14.594120+00:00 [queued]>
[2022-02-15 03:25:38,201] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.test.my_new_project.dim_customers manual__2022-02-15T03:24:14.594120+00:00 [queued]>
[2022-02-15 03:25:38,202] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:25:38,203] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-15 03:25:38,204] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:25:38,217] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): test.my_new_project.dim_customers> on 2022-02-15 03:24:14.594120+00:00
[2022-02-15 03:25:38,222] {standard_task_runner.py:52} INFO - Started process 2272 to run task
[2022-02-15 03:25:38,231] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dbt_training_bytecode', 'test.my_new_project.dim_customers', 'manual__2022-02-15T03:24:14.594120+00:00', '--job-id', '136', '--raw', '--subdir', 'DAGS_FOLDER/dbt-***.py', '--cfg-path', '/tmp/tmpzrer_4wh', '--error-file', '/tmp/tmps4vl5yux']
[2022-02-15 03:25:38,233] {standard_task_runner.py:77} INFO - Job 136: Subtask test.my_new_project.dim_customers
[2022-02-15 03:25:38,359] {logging_mixin.py:109} INFO - Running <TaskInstance: dbt_training_bytecode.test.my_new_project.dim_customers manual__2022-02-15T03:24:14.594120+00:00 [running]> on host 459d09dd7c86
[2022-02-15 03:25:38,470] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bytecode
AIRFLOW_CTX_DAG_ID=dbt_training_bytecode
AIRFLOW_CTX_TASK_ID=test.my_new_project.dim_customers
AIRFLOW_CTX_EXECUTION_DATE=2022-02-15T03:24:14.594120+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-15T03:24:14.594120+00:00
[2022-02-15 03:25:38,473] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-15 03:25:38,475] {subprocess.py:74} INFO - Running command: ['bash', '-c', '\n            dbt --no-write-json test --target dev --models dim_customers             --profiles-dir /opt/***/dbt/dbt-training-bytecode-1 --project-dir /opt/***/dbt/dbt-training-bytecode-1\n            ']
[2022-02-15 03:25:38,493] {subprocess.py:85} INFO - Output:
[2022-02-15 03:25:44,036] {subprocess.py:89} INFO - 03:25:44  Running with dbt=1.0.1
[2022-02-15 03:25:44,369] {subprocess.py:89} INFO - 03:25:44  Found 8 models, 11 tests, 0 snapshots, 0 analyses, 367 macros, 0 operations, 1 seed file, 3 sources, 0 exposures, 0 metrics
[2022-02-15 03:25:44,380] {subprocess.py:89} INFO - 03:25:44
[2022-02-15 03:25:44,384] {subprocess.py:89} INFO - 03:25:44  [WARNING]: Nothing to do. Try checking your model configs and model specification args
[2022-02-15 03:25:44,903] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-15 03:25:44,932] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dbt_training_bytecode, task_id=test.my_new_project.dim_customers, execution_date=20220215T032414, start_date=20220215T032538, end_date=20220215T032544
[2022-02-15 03:25:44,970] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-15 03:25:45,044] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
